hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFlusher.java

```diff
@@ -73,20 +73,20 @@ protected void finalizeWriter(StoreFileWriter writer, long cacheFlushSeqNum,
   /**
    * Creates the scanner for flushing snapshot. Also calls coprocessors.
    * @param snapshotScanners
-   * @param smallestReadPoint
    * @return The scanner; null if coprocessor is canceling the flush.
    */
   protected final InternalScanner createScanner(List<KeyValueScanner> snapshotScanners,
-      long smallestReadPoint, FlushLifeCycleTracker tracker) throws IOException {
```

Comment:
This remove the smallestReadPoint parameter from createScanner method, too? Change the issue title to talk about this?

Reply:
OK, thanks a lot~